### PR TITLE
refactor: Refactored key value pair component [131]

### PR DESCRIPTION
**Pull request by Astitva877**
## Description

Please do not leave this blank 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/lib/components/key-value/KeyValue.svelte [hunk 1]
```diff
@@ -301,61 +342,137 @@
...
-                    {trackParanthesis}
-                    updateText={(url) => {
-                      element.value = url;
-                    }}
-                    handleInputValue={() => {
+              {#if type === "file"}
+                <div class="flex-grow-1 w-100">
+                  <div
+                    class="position-relative rounded p-1 d-flex backgroundColor"
+                    style="height: 27px;"
```
Comment (parthdalaltech) [inline src/lib/components/key-value/KeyValue.svelte:349]: height 27px fixed any reason?

File: src/lib/components/key-value/KeyValue.svelte [hunk 2]
```diff
@@ -301,61 +342,137 @@
...
+                      top:0;
+                      left:0;
+                      right:0;
+                      bottom:-1;"
+                        placeholder=""
+                      />
+                      <div
+                        class=""
+                        style="height:18px;
+                      z-index:4;
```
Comment (parthdalaltech) [inline src/lib/components/key-value/KeyValue.svelte:396]: z-4 class

File: src/lib/components/key-value/KeyValue.svelte [hunk 3]
```diff
@@ -301,61 +342,137 @@
...
+                      right:0;
+                      bottom:-1;"
+                        placeholder=""
+                      />
+                      <div
+                        class=""
+                        style="height:18px;
+                      z-index:4; 
+                      font-size:13px;
+                    position: absolute;
```
Comment (parthdalaltech) [inline src/lib/components/key-value/KeyValue.svelte:398]: p-absolute class

File: src/lib/components/key-value/KeyValue.svelte [hunk 4]
```diff
@@ -301,61 +342,137 @@
...
+                      top:0;
+                      left:10px;"
+                      >
+                        <span style="font-size:10px; margin:4px;"
```
Comment (parthdalaltech) [inline src/lib/components/key-value/KeyValue.svelte:402]: margin class from bootstrap

**Review by parthdalaltech — CHANGES_REQUESTED** [review]

File: src/lib/components/key-value/KeyValue.svelte (hunk 2 above)
Comment (Astitva877) [reply src/lib/components/key-value/KeyValue.svelte]: Resolved.

File: src/lib/components/key-value/KeyValue.svelte (hunk 3 above)
Comment (Astitva877) [reply src/lib/components/key-value/KeyValue.svelte]: Resolved.

File: src/lib/components/key-value/KeyValue.svelte (hunk 4 above)
Comment (Astitva877) [reply src/lib/components/key-value/KeyValue.svelte]: resolved.

File: src/lib/components/key-value/KeyValue.svelte (hunk 1 above)
Comment (Astitva877) [reply src/lib/components/key-value/KeyValue.svelte:349]: Actually we have given this as per Figma, so the height of input text for file type for key value component stays the same wherever we have used this, for text type it is also the same.

**Review by parthdalaltech — APPROVED** [review]

**Pull request merged**